code/Test_definitions/location-verification.feature

```diff
@@ -115,70 +117,82 @@ Feature: CAMARA Device location verification API, v1.0.0 - Operation verifyLocat
     When the HTTP "POST" request is sent
     Then the response status code is 404
     And the response property "$.status" is 404
-    And the response property "$.code" is "DEVICE_NOT_FOUND"
+    And the response property "$.code" is "IDENTIFIER_NOT_FOUND"
     And the response property "$.message" contains a user friendly text
 
-  @location_verification_14_device_identifiers_mismatch
-  Scenario: Device identifiers mismatch
-    Given that al least 2 types of device identifiers are supported by the implementation
-    And the request body property "$.device" includes several identifiers, each of them identifying a valid but different device
+
+  @location_verification_C02.04_unnecessary_device
+  Scenario: Device not to be included when it can be deduced from the access token
+    Given the header "Authorization" is set to a valid access token identifying a device
+    And the request body property "$.device" is set to a valid device
     When the HTTP "POST" request is sent
     Then the response status code is 422
     And the response property "$.status" is 422
-    And the response property "$.code" is "DEVICE_IDENTIFIERS_MISMATCH"
-    And the response property "$.message" contains a user friendly text
+    And the response property "$.code" is "UNNECESSARY_IDENTIFIER"
+    And the response property "$.message" contains a user-friendly text
+
 
-  @location_verification_15_device_token_mismatch
-  Scenario: Inconsistent access token context for the device
-    # To test this, a token have to be obtained for a different device
-    Given the request body property "$.device" is set to a valid testing device
-    And the header "Authorization" is set to a valid access token emitted for a different device
+  @location_verification_C01.05_missing_device
+  Scenario: Device not included and cannot be deduced from the access token
+    Given the header "Authorization" is set to a valid access token which does not identify a single device
+    And the request body property "$.device" is not included
     When the HTTP "POST" request is sent
-    Then the response status code is 403
-    And the response property "$.status" is 403
-    And the response property "$.code" is "INVALID_TOKEN_CONTEXT"
-    And the response property "$.message" contains a user friendly text
+    Then the response status code is 422
+    And the response property "$.status" is 422
+    And the response property "$.code" is "MISSING_IDENTIFIER"
+    And the response property "$.message" contains a user-friendly text
+
+
+  @location_verification_C01.06_unsupported_device
+  Scenario: None of the provided device identifiers is supported by the implementation
+    Given that some types of device identifiers are not supported by the implementation
+    And the request body property "$.device" only includes device identifiers not supported by the implementation
+    When the HTTP "POST" request is sent

```

Comment:
We should probably add: And the header "Authorization" is set to a valid access token which does not identify a single device

Reply:
Yes, you are right, but this is copied from Commonalities artifact, so I will have to update there as well